components/css/content/_contentPage.scss

```diff
@@ -52,11 +52,41 @@
 					width:100%;
 					margin-bottom:20px;
 				}
-				ul{
+				ul, ol {
+					list-style-type: none;
 					li{
 						padding:3px 0;
 					}
 				}
+				ol > li, ul > li {
+					&::before {
+						width: 1em;
+						margin-left: -1em;
+						position: relative;
+						color: $color-campusGold;
+						font-size: calc-toRem(18);
+					}
+				}
+				ul > li {
+					&::before {
+						content: "•";
+						display: inline-block;
+						font-size: calc-toRem(33);
+						line-height: 0;
+						top: 5px;
```

Comment:
I may need to make these (r)em so they'll scale nicely.

Reply:
Yep! I think using calc-toRem(5) in its place should work.